packages/pluggableWidgets/maps-web/src/Maps.editorPreview.tsx

```diff
@@ -23,11 +11,11 @@ export const preview = (props: MapsPreviewProps): ReactNode => {
                     Provider unavailable without API Key, preview is not possible at the moment
                 </Alert>
             )}
-            <div style={imagePreviewStyles} />
+            <div className="widget-maps-preview" />
         </div>
     );
 };
 
 export function getPreviewCss(): string {
-    return require("./ui/Maps.css");
+    return require("./ui/MapsPreview.scss");
```

Comment:
so it means that with this method we can only require `scss`?

Reply:
I fixed the initial issue, just renamed to scss to keep the pattern as other widgets also use it